src/main/java/nl/tudelft/semgroup4/MainApp.java

```diff
@@ -61,7 +61,7 @@ public static void main(String[] args) throws SlickException {
     public void initStatesList(GameContainer container) throws SlickException {
         Resources.init();
         Settings.init();
-        addState(new StartScreenState());
+        addState(new StartScreenState(new ResourcesWrapper()));
```

Comment:
New recourses could just be set in the constructor and don't have te be given as a parameter.